lib/std/c/parse.zig

```diff
@@ -617,18 +617,18 @@ const Parser = struct {
                     return false;
                 };
                 switch (ty.id) {
-                    .Enum => |e| blk: {
+                    .Enum => |e| inner: {
```

Comment:
cc @Vexu there were some block label shadowings here. is this change correct?

Reply:
This seems correct, however this whole thing is heavily unfinished and likely bitrotten and should probably be deleted in favor of being replaced with arocc, rewritten entirely, or not being included in the std at all.